bin/chardetect

```diff
@@ -1,4 +1,4 @@
-#!/home/wavy/bennett/job-runner/.venv/bin/python
+#!/home/alice/code/job-runner/.venv/bin/python
```

Comment:
I'm assuming that this line should still go in?

Reply:
Yeah, it's fine, its just a comment.

It's an artefact of the weird way we are shipping these dependencies, basically, this repo is a a copy of the site-packages directory of a venv.